docs/conf.py

```diff
@@ -316,7 +316,11 @@ def linkcode_resolve(domain, info):
 # html_use_smartypants = True
 
 # Custom sidebar templates, maps document names to template names.
-# html_sidebars = {}
+
+html_sidebars = {
+    "**": ["sidebar-nav-bs", "sidebar-ethical-ads"],
```

Comment:
This now seems to give us the ads twice? That's perhaps overdoing thing?

Reply:
I can't replicate this in my browser :( . It shouldn't be doing so, could you attach a screenshot?

Reply:
Sure, see below
<img width="1265" alt="Screenshot 2024-07-30 at 14 34 36" src="https://github.com/user-attachments/assets/bcf10110-e95e-4cf4-b84d-c53f732beb0c">

Reply:
Ok, this is really strange. I have gone back and forth between getting 1 ad and 2 ads, even changing based on the page reload or the tab I have open. The placement of the ad is determined by a div with `id="ethical-ad-placement"`, and sometimes when reloading there are two of these divs.

I'm not sure if this is a problem on ReadTheDocs side (where it isn't properly identifying the div, so it assumes it doesn't exist and inserts its own), or if its a problem with the PyData theme and its documented defaults.

I'm going to try removing the explicit placement and see if that helps.

Reply:
10c6824 looks to have fixed it. Feel free to test it on your end @erikvansebille

Reply:
Indeed, seems fixed now